lib/utils/logger.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('lodash');
```

Comment:
As in comment bellow. Let's add this to jira ticket to not forgot about it. Thanks